docs/source/basics.rst

```diff
@@ -274,15 +274,18 @@ be executed 1000 times.
 Parametric Compilation
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Keri to improve:
-[
-Making use of declared memory regions, we can load values to parametric gates at execution time, after compilation. We
-can specify how we want to patch our compiled executable binaries with updated parameters. For some quantum algorithms,
-it's necessary to run a program repeatedly, but with different parameter values each time.
-Taking the compiler out of the execution loop for programs like this offers a huge performance improvement over compiling
-the whole program each time with those values filled in.
-(Some more details about this and an example are found :ref:`here <parametric>`.)
-]
+Modern quantum algorithms are often parametric, following a hybrid model. In this hybrid
+model, the program ansatz (template of gates) is fixed, and iteratively updated with new
+parameters. These new parameters are often determined by an update given by a classical
+optimizer. Depending on the complexity of algorithm, the problem of interest, and capabilities
```

Comment:
my mistake - small typo. "complexity of algorithm, the problem of interest" to "complexity of the algorithm, problem of interest"